spec.bs

```diff
@@ -462,21 +462,30 @@ The details for servers implementing this protocol can be found in [[!ISSUER-PRO
 Browser Steps For Issue Response {#browser-issue-response}
 ----------------------------------------------------------
 
-To process a response to an issue request, browser carries out the following steps.
+To <dfn>handle an issue response</dfn>, given [=request=] |request| and [=response=] |response|, run the following steps:
 
- 1. Let |p| be [=a new promise=].
- 1. If the response has no <a http-header>Sec-Private-State-Token</a> header, [=/reject=] |p| with an "{{InvalidStateError}}" {{DOMException}} and return |p|.
- 1. If the response has an empty <a http-header>Sec-Private-State-Token</a> header, [=/resolve=] |p| with {{undefined}} and return |p|. (This is a `Success` response bearing 0 tokens.)
- 1. Remove the <a http-header>Sec-Private-State-Token</a> header from the response and carry out the cryptographic procedures to obtain a list of unmasked tokens.
- 
- Issue: Define "cryptographic procedures to obtain a list of unmasked tokens"
-    1. If the cryptographic procedure succeeds, associate the tokens with the issuing key's label and store the tokens. [=/Resolve=] |p| with {{undefined}} and return |p|.
-    1. Else, [=/reject=] |p| with a "{{DataError}}" {{DOMException}} and return |p|.
+1. If |request|'s [=request/header list=] [=header list/contains|does not contain=] <a http-header>Sec-Private-State-Token</a>, return null.
+1. If |response|'s [=response/header list=] [=header list/contains|does not contain=] <a http-header>Sec-Private-State-Token</a>, return a [=network error=].
+1. Let |header| be the result of [=header list/get|getting=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
+1. If |header| is empty, return null.
+1. [=header list/delete|Delete=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
+1. Let |unmasked tokens| be the result of carrying out the cryptographic procedures to obtain a list of unmasked tokens.
+1. If |unmasked tokens| is an error, return a [=network error=].
+1. Associate |unmasked tokens| with the issuing key's label and store the tokens.
+1. Return null.
 
-Issue: Define this in terms of fetch
+Issue: Define "cryptographic procedures to obtain a list of unmasked tokens"
 
 The details for servers implementing this protocol can be found in [[!ISSUER-PROTOCOL]].
 
+<h3 id="modification-to-http-fetch-steps-header">Modification to HTTP fetch steps</h3>
+The following steps will be added to the [=HTTP fetch=] steps, before checking the redirect status (i.e. "If |actualResponse|'s status is a redirect status, ..."):
```

Comment:
In general in this spec we've added the number of the step that we're monkeypatching before as well, it can be helpful in some situations. Yes, it can get out of date, but that's just a general risk with monkeypatching. I'd argue that an out-of-date step number is actually helpful as it indicates to the reader that something changed in the original spec.

Reply:
SGTM, done.